src/components/triggersAndActions.ts

```diff
@@ -122,7 +122,7 @@ export async function executeAction(
   actionLibrary: ActionLibrary
 ): Promise<boolean> {
   // TO-DO: If Scheduled, create a Job instead
-  const actionResult = actionLibrary[payload.code](payload.parameters)
+  const actionResult = await actionLibrary[payload.code](payload.parameters)
```

Comment:
Added this as result was not being written to `action_queue`. Needs to wait for result to resolve.